src/test/java/core/basesyntax/RegistrationTests.java

```diff
@@ -0,0 +1,155 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationIllegalArgumentException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationTests {
+    public static final String ANSI_YELLOW = "\u001B[33m";
+    public static final String ANSI_CYAN = "\u001B[36m";
+    public static final String ANSI_RESET = "\u001B[0m";
+    private static final long ID1 = 1L;
+    private static final long ID2 = 2L;
+    private static final int NON_ACCEPTABLE_AGE = 9;
+    private static final int ACCEPTABLE_AGE = 34;
+    private static final String EMPTY_LOGIN = "";
+    private static final String CORRECT_LOGIN = "Ukraine";
+    private static final String INCORRECT_PASSWORD = "34df";
+    private static final String CORRECT_PASSWORD = "GloryOfUkraine";
+    private static RegistrationServiceImpl registrationService;
+    private static StorageDaoImpl storageDao;
+    private static User validUser = new User();
+    private static User invalidUser = new User();
+
+    @BeforeAll
+    public static void setUp() {
+        System.out.println(ANSI_YELLOW
+                + "----------------------  Tests started  ----------------------" + ANSI_RESET);
+        storageDao = new StorageDaoImpl();
+        registrationService = new RegistrationServiceImpl();
+        validUser.setId(ID1);
```

Comment:
do not set the id to the user manually this is what a registration service does for us

Reply:
amended